torch/_inductor/scheduler.py

```diff
@@ -3072,14 +3209,23 @@ def can_fuse(self, node1: BaseSchedulerNode, node2: BaseSchedulerNode) -> bool:
         del device2
 
         shared_data_score = self.score_fusion_memory(node1, node2)
-        if shared_data_score == 0:
+        loop_ordering_log.debug(
+            "%s and %s has%s shared data before loop reordering: %d",
+            node1.get_name(),
+            node2.get_name(),
+            " no" if shared_data_score == 0 else "",
+            shared_data_score,
+        )
+
+        if shared_data_score < config.score_fusion_memory_threshold:
```

Comment:
Unrelated to this pr, it's to fix this test error, https://www.internalfb.com/intern/test/844425111960859?ref_report_id=0